BuiltDifferentMobileApp/BuiltDifferentMobileApp/ViewModels/Profile/MyProfilePageClientViewModel.cs

```diff
@@ -68,19 +68,24 @@ public bool IsEnabled
         private INetworkService<HttpResponseMessage> networkService = NetworkService<HttpResponseMessage>.Instance;
 
         public MyProfilePageClientViewModel() {
+            GetUserInfo();
+
+            SubmitCommand = new AsyncCommand(Submit);
+            UploadImageCommand = new AsyncCommand(Upload);
+            EditProfileCommand = new AsyncCommand(Edit);
+
+            isEnabled = false;
+        }
+
+        private async Task GetUserInfo()
+        {
             Models.Client user = (Models.Client)accountService.CurrentUser;
             Id = user.id;
```

Comment:
From what I understand, GetUserInfo() retrieves the CurrentUser from the account service and sets the inputs on the screen to it's properties. But I don't see anything updating the current user stored in the AccountService.
Consider fetching the current user's profile after updating it and and setting it to .CurrentUser in AccountService and then setting the form fields.
Pseudo code:

`var userInfo = networkService.GetAsync<>(CURRENT_USER_PROFILE_URI);`
// you can check how I set the current user in the network service register function
`accountService.CurrentUser = userInfo;`

Let me know if I'm wrong.

Reply:
changes have been made to reflect these suggestions. Thank you so much !

Reply:
You were correct, the changes have been rectified.